### ISSUE TITLE
API support for users lists && their members?

### ISSUE THREAD

**Issue by awitherow** [dghubble/go-twitter#78]
Hello @dghubble and other maintainers of `go-twitter`!

On Twitter, a useful feature I employ to keep track of specific niches is the Lists functionality.

The specific API call that I want to make via this library is to get the statuses of all list members. I am using this in a bot I am helping make. It posts new tweets from a users timeline at the moment, and I want to filter those posts to make sure they're only users from the list. I found this API doc below that would allow me to skip that filtering mechanism and directly get statuses only from those list members.

https://dev.twitter.com/rest/reference/get/lists/statuses

I was wondering if there is any pre-existing support in this library (did not see anything in the docs), and if there are plans to integrate such functionality?

General list support would be great 👍  and I would love to help implement this.

**Comment by awitherow** [comment]
@dghubble ping

**Comment by dghubble** [comment]
I didn't add the Lists API with the other timelines, it was new at the time. It should be fairly straight forward to add if you've got time.

**Comment by dghubble** [comment]
Actually, that's collections that I'm thinking of

**Comment by awitherow** [comment]
Yeah sure I can go for it @dghubble I just wanted to make sure it was aligned with your goals of `go-twitter`.

Could you psuedo code how you'd like it to function (something real quick) just so that I know I am making it as to your desired spec? 🙇 🙏 thanks!

**Comment by cdornsife** [comment]
I started implementing the lists endpoint. I no longer have time to work on it. It can be found here: https://github.com/cdornsife/go-twitter/blob/master/twitter/lists.go
I can submit a PR so someone else can keep working on it. This should get them started.

**Comment by randytorres** [comment]
@cdornsife I have some time over the next few days to finish up the pr

**Comment by cdornsife** [comment]
Ok, ill submit a [WIP] Pr that you (@randytorres) can continue working on.
